src/building/monument.c

```diff
@@ -1150,3 +1150,21 @@ void building_monument_delivery_load_state(buffer* list)
 		delivery_load(list, i);
 	}
 }
+
+int building_monument_is_construction_halted(const building* b) {
```

Comment:
Stylistic comment - pointer symbol should be attached to the variable name, not the type.
I.e. `int *variable` not `int* variable`.

Reply:
true. i just happen to notice `int* variable` is the case for functions above.